docs/en/ingest-management/security/certificates.asciidoc

```diff
@@ -231,6 +231,7 @@ sudo ./elastic-agent install \
    --fleet-server-port=8220 \
    --elastic-agent-cert=/tmp/fleet-server.crt \
    --elastic-agent-cert-key=/tmp/fleet-server.key \
+   --elastic-agent-cert-key-passphrase=/tmp/fleet-server ???? \
```

Comment:
Probably a dumb question, but what would the path actually look like? I wasn't sure what file format the passphrase would have.

Reply:
The passphrase file will just be a plain test file, so what you have is fine.